src/backend/tests/risks.test.ts

```diff
@@ -0,0 +1,45 @@
+import request from 'supertest';
+import express from 'express';
+import riskRouter from '../src/routes/risks.routes';
+
+const app = express();
+app.use(express.json());
+app.use('/', riskRouter);
+
+const editRiskTrue = {
+  userId: 1,
+  id: '4800c304-2f8d-4376-a4c6-d7a157134c83',
+  detail: 'This one might be a bit too expensive',
+  resolved: true
+};
+
+const editRiskFalse = {
+  userId: 1,
+  id: '4800c304-2f8d-4376-a4c6-d7a157134c83',
+  detail: 'This one might be a bit too expensive',
+  resolved: false
+};
```

Comment:
move these to a test-data file and import them into this test file

Reply:
👍🏽